examples/tabs/css/tabs.css

```diff
@@ -1,107 +1,67 @@
 .tabs {
-  width: 20em;
   font-family: "lucida grande", sans-serif;
 }
 
 [role="tablist"] {
-  margin: 0 0 -0.1em;
-  overflow: visible;
+  min-width: 550px;
 }
 
-[role="tab"] {
+[role="tab"],
+[role="tab"]:focus,
+[role="tab"]:hover {
   position: relative;
+  z-index: 2;
+  top: 2px;
   margin: 0;
-  padding: 0.3em 0.5em 0.4em;
+  padding: 3px 3px 4px;
   border: 1px solid hsl(219deg 1% 72%);
-  border-radius: 0.2em 0.2em 0 0;
-  box-shadow: 0 0 0.2em hsl(219deg 1% 72%);
+  border-bottom: 2px solid hsl(219deg 1% 72%);
+  border-radius: 5px 5px 0 0;
   overflow: visible;
-  font-family: inherit;
-  font-size: inherit;
   background: hsl(220deg 20% 94%);
-}
-
-[role="tab"]:hover::before,
-[role="tab"]:focus::before,
-[role="tab"][aria-selected="true"]::before {
-  position: absolute;
-  bottom: 100%;
-  right: -1px;
-  left: -1px;
-  border-radius: 0.2em 0.2em 0 0;
-  border-top: 3px solid hsl(20deg 96% 48%);
-  content: "";
+  outline: none;
+  font-weight: bold;
 }
 
 [role="tab"][aria-selected="true"] {
-  border-radius: 0;
+  padding: 2px 2px 4px;
+  border-width: 2px;
+  border-bottom-color: hsl(220deg 43% 99%);
```

Comment:
I don't understand any of these CSS changes? They don't seem to relate to the requested changes?

Reply:
To support high contrast modes I am trying to emphasize at least 2 pixel spacing and borders to support operating system high contrast modes for focus styling.  This is easier for developers to see if pixels are used in the code.

Reply:
Ah I see what you’re saying now. But wouldn’t a transparent outline achieve the same for high contrast mode? Would’ve been a far less invasive change.